source/grid/grid_generator_from_name.cc

```diff
@@ -281,6 +281,27 @@ namespace GridGenerator
       return false;
     }
 
+    /**
+     * Methods implemented only for Triangulation<2,2>.
+     *
+     * Return true if a grid was actually generated, false otherwise.
+     */
+    bool
+    generate_special(const std::string &  name,
+                     const std::string &  arguments,
+                     Triangulation<2, 2> &tria)
```

Comment:
We only need this because this function is not instantiated in 1D, right? I'd prefer to instantiate a 1D stub that throws an exception than define this since that's what quarter_hyper_ball, half_hyper_ball, etc. do.